### PR TITLE
dark forest thoughts for warriors, leaders, and apprentices

### PR DESCRIPTION
**Pull request by selkirks**
added various dark forest for warriors, leaders, and apprentices! noticed they didnt have any so i went a tad bit silly

### CODE REVIEW

File: resources/dicts/thoughts/dead/darkforest/apprentice.json [hunk 1]
```diff
@@ -1,3 +1,19 @@
 [
-    
+	{
+		"id": "apprentice_dark_forest_thoughts",
+		"thoughts": [
+			"Wishes they finished their battle training",
```
Comment (Lixxis) [inline resources/dicts/thoughts/dead/darkforest/apprentice.json]: Here is the PRONOUN tag is missing.

File: resources/dicts/thoughts/dead/darkforest/apprentice.json [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+			"Tries to imitate the battle moves of other Dark Forest cats",
+			"Wonders if it is {PRONOUN/m_c/poss/} fault or {PRONOUN/m_c/poss} mentor's fault that they ended up here",
+			"Wishes they could put their hunting training to use",
```
Comment (Lixxis) [inline resources/dicts/thoughts/dead/darkforest/apprentice.json]: Here is the PRONOUN tag is missing.

File: resources/dicts/thoughts/dead/darkforest/apprentice.json [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+			"Misses the elders' stories",
+			"Regrets ever disrespecting their mentor",
```
Comment (Lixxis) [inline resources/dicts/thoughts/dead/darkforest/apprentice.json]: Here is the PRONOUN tag is missing.

**Review by Lixxis — CHANGES_REQUESTED** [review]

File: resources/dicts/thoughts/dead/darkforest/apprentice.json (hunk 3 above)
Comment (selkirks) [reply resources/dicts/thoughts/dead/darkforest/apprentice.json]: fixed pronoun tag issues!

**Review by Lixxis — APPROVED** [review]

File: resources/dicts/thoughts/dead/darkforest/apprentice.json (hunk 1 above)
Comment (Lixxis) [reply resources/dicts/thoughts/dead/darkforest/apprentice.json]: I think this is still wrong?

**Pull request merged**